react-frontend/src/components/UserFeedback/feedback.js

```diff
@@ -46,7 +46,11 @@ export default function FeedbackForm() {
 
   return (
     <>
-      <ModalButton type="primary" onClick={showModal}>
+      <ModalButton
+        type="primary"
+        onClick={showModal}
+        alt="Provide feedback on this site"
```

Comment:
Sorry, when writing up #899 I did not realize that `alt` doesn't work on a `<button>` ... but `aria-label` does ...
If you change `alt` here to `ariaLabel`, the screen reader will read out "Provide feedback on this site"